smartdashboard/views.py

```diff
@@ -341,3 +345,467 @@ def __init__(
         self.app_view = app_view
         self.ens_view = ens_view
         self.orc_view = orc_view
+
+    def update(self) -> None:
+        """Update all views within the OverviewView"""
+        self.exp_view.update()
+        self.app_view.update()
+        self.ens_view.update()
+        self.orc_view.update()
+
+
+@dataclass(frozen=True)
+class Files:
+    graph_file: str
+    table_file: str
```

Comment:
I'm on the fence about this one only because all the paths I'm getting from the `manifest.json` file are strings. There'd be sooooo many pathlib.Path casts everywhere.

Reply:
I'm willing to defer to your expertise over the codebase! If its more hassle to cast the higher order type everywhere than the convenience of the higher order type is worth, I think it seems reasonable to keep the `str`s